search/api_test.py

```diff
@@ -96,11 +99,13 @@ def test_execute_search(self):
             assert search_obj.execute.called
             assert mock_get_conn.called
 
-    def test_create_search_obj_filter(self):
+    @ddt.data(True, False)
+    def test_create_search_obj_filter(self, is_advance_search_capable):
         """
         Test that Search objects are created with program-limiting and filled_out=True query parameters
         """
-        search_obj = create_search_obj(self.user)
+        user = self.user if is_advance_search_capable else self.learner
+        search_obj = create_search_obj(user)
```

Comment:
I'm actually already asserting this in the test below `test_create_search_obj_metadata`, do you think this needs to be asserted here as well?